plugins/CoreAdminHome/Commands/PurgeOldArchiveData.php

```diff
@@ -68,7 +69,12 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $archivePurger = $this->archivePurger;
+        $logger = null;
+        if ($output->getVerbosity() <= OutputInterface::VERBOSITY_NORMAL) {
+            $logger = new NullLogger();
+        }
```

Comment:
During normal output (in addition to when `-q` is used), we don't want the ArchivePurger logs to output in this command, but we want it to appear in scheduled task output.